docs/design/feedback.rst

```diff
@@ -0,0 +1,119 @@
+.. vale off
+
+Providing effective user feedback
+=================================
+
+.. vale on
+
+When developing features, it's crucial to ensure that Users receive clear feedback and guidance when certain information or data isn't available. Instead of simply hiding tabs or displaying zeroed metrics, for example, adopt a proactive approach to inform and guide Users.
+
+Fundamental principles
+----------------------
+
+The principles of visibility, transparency, and guidance Form the foundation of an intuitive and informative User experience.
+
+- **Visibility**: keep all functionalities visible, even when there is no data available.
+- **Transparency**: communicate that information is missing.
+- **Guidance**: provide instructions on how to obtain or enable the necessary data.
+
+Visibility ensures that Users are aware of all available functionalities, even when they're not active or populated. Transparency builds trust by clearly explaining why certain information might be missing. Guidance empowers Users by providing clear paths for action and improvement. Together, these principles transform Points of frustration into solutions, helping marketing professionals complete their tasks.
```

Comment:
```suggestion
Visibility ensures that Users are aware of all available functionalities, even when they're not active or populated. Transparency builds trust by clearly explaining why certain information might be missing. Guidance empowers Users by providing clear paths for action and improvement. Together, these principles transform moments of frustration into solutions, helping marketing professionals complete their tasks.
```
No need to capitalise points here, we're not referring to the Mautic feature. Might be better to use moments?

Reply:
Great, makes sense